VimeoUpload/Descriptor System/ProgressDescriptor.swift

```diff
@@ -26,7 +26,7 @@
 
 import Foundation
 
-public class ProgressDescriptor: Descriptor
+open class ProgressDescriptor: Descriptor
```

Comment:
can we change all these opens to public?  Do we subclass ProgressDescriptor anywhere?